scripts/opencontrail_control_install.sh

```diff
@@ -7,9 +7,14 @@ salt -C 'I@opencontrail:database' state.sls opencontrail.database -b 1
 salt -C 'I@opencontrail:control' state.sls opencontrail -b 1
 
 # Provision opencontrail control services
-salt -C 'I@opencontrail:control:id:1' cmd.run "/usr/share/contrail-utils/provision_control.py --api_server_ip 172.16.10.254 --api_server_port 8082 --host_name ctl01 --host_ip 172.16.10.101 --router_asn 64512 --admin_password workshop --admin_user admin --admin_tenant_name admin --oper add"
-salt -C 'I@opencontrail:control:id:1' cmd.run "/usr/share/contrail-utils/provision_control.py --api_server_ip 172.16.10.254 --api_server_port 8082 --host_name ctl02 --host_ip 172.16.10.102 --router_asn 64512 --admin_password workshop --admin_user admin --admin_tenant_name admin --oper add"
-salt -C 'I@opencontrail:control:id:1' cmd.run "/usr/share/contrail-utils/provision_control.py --api_server_ip 172.16.10.254 --api_server_port 8082 --host_name ctl03 --host_ip 172.16.10.103 --router_asn 64512 --admin_password workshop --admin_user admin --admin_tenant_name admin --oper add"
+hosts=(`salt-call pillar.get linux:network:host | egrep 'ctl0.*:' | sed -e 's/  *//' -e 's/://'`)
```

Comment:
I see that you use the same pattern to get the list of minion hosts. This might be worth factoring this into a function.